app/controllers/api/v1/reviews_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::ReviewsController < ApplicationController
+
+  def show
+    stop = Stop.find(params[:stop_id])
+    render json: stop.reviews
+  end
```

Comment:
this is actually either an `index` path for Reviews, or a `show` path for Stops. I would probably move this to a Api::V1::StopsController#show instead. 

The data should be loaded on the ComponentDidMount of your StopShow container

Reply:
Woops saw your other code. If this endpoint isnt being removed then, then just delete it.